src/SparklinesCurve.js

```diff
@@ -3,72 +3,63 @@ import React from 'react';
 
 export default class SparklinesCurve extends React.Component {
 
-    static propTypes = {
-        color: PropTypes.string,
-        style: PropTypes.object
-    };
-
-    static defaultProps = {
-        style: {}
-    };
+  static propTypes = {
+    color: PropTypes.string,
+    style: PropTypes.object
+  };
 
-    render() {
-        const { points, width, height, margin, color, style, divisor = 0.25 } = this.props;
-        let prev;
-        const curve = (p) => {
-            let res;
-            if (!prev) {
-                res = [p.x, p.y]
-            } else {
-                const len = (p.x - prev.x) * divisor;
-                res = [ "C",
-                    //x1
-                    prev.x + len,
-                    //y1
-                    prev.y,
-                    //x2,
-                    p.x - len,
-                    //y2,
-                    p.y,
-                    //x,
-                    p.x,
-                    //y
-                    p.y
-                ];
-            }
-            prev = p;
-            return res;
+  static defaultProps = {
+    style: {}
+  };
 
-        }
-        const linePoints = points
-            .map((p) => curve(p))
-            .reduce((a, b) => a.concat(b));
```

Comment:
This creates a bunch of thrown-away arrays that will make the garbage collector perform a bunch of needless work.